Makefile

```diff
@@ -229,6 +229,7 @@ test-proto-static:
 
 proto: docker-build-proto
 	./etc/proto/build.sh
+	./python-sdk/proto/generate-protos.sh
```

Comment:
I don't know much about Makefiles or if you can chain `make` calls, but should we try to hook into the command within python-sdk/Makefile and run `make -C python-sdk proto`?